editor/modes/visual-editor/style.scss

```diff
@@ -255,3 +255,39 @@ $sticky-bottom-offset: 20px;
 		content: '';
 	}
 }
+
+.editor-visual-editor__placeholder {
```

Comment:
>I don't like extracting shared styles just because they are the same.

Okay, I'd not recommend it if they're not truly "shared". I thought we were emulating block effects by duplicating them here. We can leave it as-is.